instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
@bradenmacdonald I don't feel the permission really belongs on a model, since it controls access to more than just that model.  The fact that it happens to be easy to create model permissions doesn't seem a strong enough reason to put the permission in the wrong place, and I would prefer the current solution.  (What do you mean by "global model", by the way?)

The custom model is just a proxy model used to encapsulate creating the content type you need for a permission.  It would be perfectly possible to use the `Permission` model directly.